aiven/client/cliarg.py

```diff
@@ -132,6 +132,7 @@ def wrapped(self: CommandLineTool) -> T:
     help="Minimum required nodes In Sync Replicas (ISR) to produce to a partition (default: 1)",
 )
 arg.organization_id = arg("--organization_id", help="Organization identifier")
+arg.positional_group_id = arg("group_id", help="Organization user group identifier")
```

Comment:
Arg names should use `-` instead of `_`

```suggestion
arg.positional_group_id = arg("group-id", help="Organization user group identifier")
```